### PR TITLE
Add definition of 'actual error' (Spanish)

### PR DESCRIPTION
**Pull request by JorgeBornemann**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Jorge Bornemann

## Language: 

- Spanish

## Terms defined:

- actual error
- 
-

### CODE REVIEW

**Comment by JorgeBornemann** [comment]
## Further definitions:
* expected_result
* pass_test
* fail_test

**Comment by ian-flores** [comment]
@JorgeBornemann It seems like one of the terms is missing in the YAML file, can you take a look, please?

File: glossary.yml [hunk 1]
```diff
@@ -3625,7 +3636,10 @@
     term: መውደቅ (ፈተና)
     def: >
       [ትክክለኛው ውጤት](#actual_result) የማይዛመድ ከሆነ አንድ ሙከራ አልተሳካም የተጠበቀው ውጤት (#expected_result)።
-
+  es:
+    term: "fallar (una prueba)"
+    def: >
+      Una prueba falla si el [resultado real](#actual_result) no coincide con el [resultado esperado](#expected result).
```
Comment (ian-flores) [inline glossary.yml:3642]: It seems like the error is in this line. #expected_result should be the text

**Review by ian-flores — CHANGES_REQUESTED** [review]

**Comment by JorgeBornemann** [comment]
Sorry, typo missing underscore. corrected in commit [cbc2fbd](https://github.com/carpentries/glosario/pull/470/commits/cbc2fbdcc1fd4706342b2e498c77308051fccb27)

**Comment by ian-flores** [comment]
Okay, fixed now, thank you!

**Review by ian-flores — APPROVED** [review]

**Comment by elletjies** [comment]
@JorgeBornemann Thank you for contributing to Glosario in 2022! We are looking forward to receiving more PRs from you during 2023!

**Pull request merged**